packages/driver/src/cypress/browser.js

```diff
@@ -0,0 +1,26 @@
+const _ = require('lodash')
+const $utils = require('./utils')
+
+const isBrowser = function (config, obj = '') {
```

Comment:
hmm, i never noticed that `obj = ''` here, this means `isBrowser()` will just return false :thinking: